CHANGELOG.md

```diff
@@ -17,11 +17,16 @@ should change the heading of the (upcoming) version to include a major version b
 -->
 # v5.0.0 (coming soon)
 
-# v4.0.2 (upcoming)
+# v4.0.3 (upcoming)
+
+# v4.0.2
 
 ## @rjsf/core
 
 - To improve performance, skip validating subschemas in oneOf / anyOf if formData is undefined (#2676)
+- Fixed the `toIdSchema()` typescript definition to add new `idSeparator` prop
+  - Also passed the new `idSeparator` prop through to the `AnyOfField` and `OneOfField` inside of `SchemaField` 
+  - Updated `ArrayField` in `@rjsf/core` to pass `idSeparator` and `idPrefix` through to `SchemaField`
```

Comment:
Can you update this line to make it clearer that a bug was fixed?

Reply:
@heath-freenome I think you forgot to do this

Reply:
> @heath-freenome I think you forgot to do this

It is there... the `idPrefix={this.props.idPrefix}` and `idSeparator={this.props.idSeparator}`

Reply:
I meant, "update this line to make it clearer that a bug was fixed"